Added embed content to html (#456)
Codecademy/docs: content/html/concepts/tags/terms/embed/embed.md

---
Title: '<embed>'
Description: 'Inserts a video, image, page, and more.'
Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'Images'
  - 'Video'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

> The `<embed>` tag allows you to embed content onto your page. It can be used to embed many types of content, however, there are more specific tags such as `<video>`, `<img>` and `<iframe>`. It should only be used if there is no better option.

Reviewer: ```suggestion
The `<embed>` tag renders an element that wraps around interactive content, including images, videos, and even other web pages. This is achieved with tools such as browser plug-ins. However, since most modern browsers no longer support plug-ins, there are other tags to use instead of `<embed>`. This includes:

- Playing video content with the [`<video>`](https://www.codecademy.com/resources/docs/html/tags/video) tag.
- Displaying images with the [`<img>`](https://www.codecademy.com/resources/docs/html/tags/img) tag.
- Rendering additional markup with the [`<iframe>`](https://www.codecademy.com/resources/docs/html/tags/iframe) tag. 
```

---
Title: '<embed>'

> Description: 'The <embed> tag allows you to "Embed" a video, image, page and more.'

Reviewer 2: ```suggestion
Description: 'The <embed> tag allows one to insert a video, image, page and more.'
```

> Description: 'The <embed> tag allows you to "Embed" a video, image, page and more.'

Reviewer 3: ```suggestion
Description: 'Inserts a video, image, page, and more.'
```

For all the Terms entries in Docs, the `Description` field in the metadata should start with the verb.

Also added an Oxford comma.

Subjects:
  - 'Web Development'
  - 'Web Design'
Tags:
  - 'Images'
  - 'Video'

>   - 'HTML'

Reviewer 3: Take out line 10 (`- 'HTML'`)

CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

> The `<embed>` tag allows you to embed content onto your page. It can be used to embed many types of content, however there are more specific tags such as `<video>`, `<img>` and `<iframe>`. It should only be used if there is no better option

Reviewer 2: ```suggestion
The `<embed>` tag allows you to embed content onto your page. It can be used to embed many types of content, however, there are more specific tags such as `<video>`, `<img>` and `<iframe>`. It should only be used if there is no better option.
```

> <!--  -->

Reviewer 2: ```suggestion
```

## Syntax

```html

> <embed type="image/jpg" src="my_image.jpg" width="300" height="200" />

Reviewer 2: Maybe we can explain the attributes in further detail here.

> ```

Reviewer 2: Can even add a **Note:** here that this tag is deprecated so we are covering it for legacy purposes.

Reviewer 2: It would also be great to add that this is a self-closing tag (also referred to as void or empty tags).

> ```

Reviewer 3: Optional: Add in a whole HTML file that uses the `<embed> element.

---
Title: '<embed>'

> Description: 'Inserts a video, image, page, and more.'

Reviewer 2: ```suggestion
Description: 'Inserts external content such as a video, image, page, and more.'
```
